qemu/target-i386/cpu.h

```diff
@@ -699,7 +699,7 @@ typedef enum {
 
 typedef struct SegmentCache {
     uint32_t selector;
-    target_ulong base;
+    uint64_t base;   /* handle 32 or 64 bit CPUs */
```

Comment:
Yes of course we can include that, but it would be the only instance (though I did not look too hard) I can find of a file under qemu including a file from include/unicorn.

I assumed, apparently wrongly, that one of your design principles was to keep things under qemu independent of things outside of qemu. If that is not the case, then this is not a problem.